src/convert/metadataConverter.ts

```diff
@@ -203,4 +202,13 @@ export class MetadataConverter {
     }
     return packagePath;
   }
+
+  private convertTypeToManifest(manifestFileName: DestructiveChangesType): string {
+    if (manifestFileName === DestructiveChangesType.POST) {
+      return MetadataConverter.DESTRUCTIVE_CHANGES_POST_XML_FILE;
+    } else if (manifestFileName === DestructiveChangesType.PRE) {
+      return MetadataConverter.DESTRUCTIVE_CHANGES_PRE_XML_FILE;
+    }
+    return MetadataConverter.PACKAGE_XML_FILE;
+  }
```

Comment:
Bunch of nits:
1. I don't like the name of this method.  "Type" much more commonly refers to a metadata type in SDR.  This overloads the term which affects maintainability.  This is really about getting the destructive changes file name only right?
2. I don't think it's possible for `PACKAGE_XML_FILE` to ever be returned with how it's being called in the class.
3. The name of the arg should probably be something like, `destructiveChangesType`.